doc/modules/ROOT/pages/quickstart.adoc

```diff
@@ -9,50 +9,70 @@ such as distributed maps, queues, topics and more. All, with enterprise level se
 
 {description}
 
+In this tutorial, you will complete the following steps:
+
+. Install Hazelcast .NET client
+. Start a Hazelcast cluster with single server
+. Connect the client to the cluster
+. Create and manipulate a Hazelcast map
+. Run an SQL query on the map
+. Monitor the client on Hazelcast Management Center
+
 == Before You Begin
 
 To complete this tutorial, you need the following:
 
-* The Hazelcast .NET client is distributed as a NuGet package which targets .NET Standard versions 2.0 and 2.1.
+[cols="1a,1a"]
+|===
+|Prerequisites|Useful resources
+
+|&#46;NET Framework and Core
+|Hazelcast .NET client is distributed as a NuGet package which targets .NET Standard versions 2.0 and 2.1.
 It can therefore be used in any application targeting .NET versions that support these .NET Standard versions:
-** https://dotnet.microsoft.com/en-us/download/dotnet-framework[.NET Framework] 4.6.2 and newer, on Windows
-** .NET Core https://dotnet.microsoft.com/en-us/download/dotnet/2.1[2.1] and https://dotnet.microsoft.com/en-us/download/dotnet/3.1[3.1], on Windows, Linux and MacOS
-* https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-7.2[PowerShell] shell and scripting language.
 
-The upcoming .NET 5 version supports .NET Standard 2.1, and therefore should execute the Hazelcast .NET client without issues, but that is not supported yet.
+* https://dotnet.microsoft.com/en-us/download/dotnet-framework[.NET Framework] 4.6.2 and newer, on Windows
```

Comment:
```suggestion
* https://dotnet.microsoft.com/en-us/download/dotnet-framework[.NET Framework] 4.6.2 or 4.8, on Windows
```
Although we target netstandart2.0, we don't target 4.7. I've never tried with 4.7. So, it's better say 4.6.2 or 4.8.